sites/website/src/docs/community/build-and-test.md

```diff
@@ -3,4 +3,97 @@ id: build-and-test
 title: Build and Test
 sidebar_label: Build and Test
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website/src/docs/community/build-and-test.md
----
\ No newline at end of file
+---
+
+# Testing
+
+## Checklist
+
+This testing checklist is a good starting point for manual testing web applications.
+
+1. Test each page for design and content issues
+    1. Test screen sizes from viewport 1 to 6
+    2. Test screen orientations
+        1. Portrait
+        2. Landscape
+    3. Test word length and case
+        1. Test the smallest amount of text
+        2. Test the largest amount of text
+        3. Case sensitivity
+2. Test for localization (RTL, LTR)
+3. Test for progressive enhancement
+    1. Test complete browser matrix
+    2. Test for zooming browser from 200% and 25%
+4. Test each component and pages for accessibility issues
+    1. Test for keyboard accessibility
+    2. Test for Mouse only interaction
+    3. Test for Touch interaction
+    4. Test with High Contrast of OS
+    5. Test without images
+    6. Test for text only
+    7. Test with a screen reader
+
+## Manual testing
+
+FAST UI development tools offer a solution for manually testing components during development. Easily test with features such as viewport, localized direction (RTL or LTR), background transparency, color themes, and interactive properties.
+
+Additionally, we offer testing services under `./build/testing` that can be used to assist in comprehensive testing. We have included these configurations and tests as a starting point. Run the tests manually, and the cloud services dashboard can be used to evaluate your code changes before submitting for code review.
+
+### Testing on physical devices
+
+When testing local code on physical devices, Localtunnel allows you to easily share a web service on your local machine without managing DNS and firewall settings.
+
+Localtunnel assigns a unique publicly accessible URL that proxies all requests to your local web server.
+
+#### Quickstart
+
+Install Localtunnel globally (requires NodeJS):
+
+```bash
+npm install -g localtunnel
+```
+
+* Start a web server on a local port (eg, [http://localhost:8000](http://localhost:8000)) and use the command line interface to request a tunnel to the local server:
+
+```bash
+lt --port 8000
+```
+
+* You receive a URL to share as long as your local instance remains active. Requests route to your local service at the specified port.
+
+### Testing with Sauce Labs
```

Comment:
Does this actually work? Do we have a sauce-labs account?

Reply:
Yeah.  Additionally, I just logged in with my GitHub account and looks like Screener.io is now part of Sauce Labs.  I know quite a few internal teams use Screener.io https://screener.io/

Reply:
When people are ready to use this, simply send me a request and I can invite you to our organization account.  The comments in code explain how to configure user/key as env variables to integration and run the script.

Reply:
Shouldn't that be part of this content?

Reply:
This feels like a disconnect if we're telling people to build and test using these steps but we first have to add them to our organization. Will we add *anyone* to that organization? Are these steps really intended for the community?

Sorry if I'm just missing something here.

Reply:
I can see that perspective.  I've moved the Sauce Labs docs off of the site docs and into `./build/testing` with the rest of the SL details.